packages/amplication-client/src/Components/PanelCollapsible.tsx

```diff
@@ -8,8 +8,14 @@ import "./PanelCollapsible.scss";
 
 type Props = {
   onCollapseChange?: (open: boolean) => {};
+  /**Whether the panel is initially open or not */
   initiallyOpen?: boolean;
+  /**When true the user cannot collapse manually, and the collapse button is hidden  */
+  disableManualCollapse?: boolean;
+  /**Whether the panel collapse functionality is enabled. When false, only the header of the panel is visible and the body is hidden */
+  /**By settings initiallyOpen to True, and disableManualCollapse to True, the developer can control the collapse state from the parent component by changing enableCollapse */
```

Comment:
SHould be in the same doc comment.
```suggestion
  /**
   * Whether the panel collapse functionality is enabled. When false, only the header of the panel is visible and the body is hidden.
   * By settings initiallyOpen to True, and disableManualCollapse to True, the developer can control the collapse state from the parent component by changing enableCollapse
   */
```